sdks/python/apache_beam/yaml/yaml_mapping.py

```diff
@@ -661,14 +669,32 @@ def normalize_fields(pcoll, fields, drop=(), append=False, language='generic'):
 
 @beam.ptransform.ptransform_fn
 @maybe_with_exception_handling_transform_fn
-def _PyJsMapToFields(pcoll, language='generic', **mapping_args):
+def _PyJsMapToFields(
+    pcoll,
+    fields: Mapping[str, Union[str, Mapping[str, str]]],
+    append: bool = False,
+    drop: Iterable[str] = (),
+    language: str = 'generic'):
   """Creates records with new fields defined in terms of the input fields.
 
   See more complete documentation on
   [YAML Mapping Functions](https://beam.apache.org/documentation/sdks/yaml-udf/#mapping-functions).
+
+  Args:
+    fields: The output fields to compute, each mapping to the expression or
+      callable that creates them.
+    append: (Optional) Whether to append the created fields to the set of
+      fields already present, outputting a union of both the new fields and
+      the original fields for each record.  Defaults to False.
+    drop: (Optional) If `append` is true, enumerates a subset of fields from the
+      original record that should not be kept
+    language: (Optional) The language used to define (and execute) the
+      expressions and/or callables in `fields`. Defaults to generic.
+    error_handling: (Optional) Whether and where to output records that throw errors when
```

Comment:
The `(Optional)` tags should already be added by `generate_yaml_docs` since all these args have default values (which would cause it to get printed twice)